src/Jackalope/Node.php

```diff
@@ -1564,12 +1575,24 @@ public function getIterator()
      * @see Node::refresh
      * @see Node::__construct
      */
-    protected function _setProperty($name, $value, $type, $internal)
+    protected function loadProperty($name)
     {
+        if (isset($this->properties[$name])) {
+            return $this->properties[$name];
+        }
+
+        if (!isset($this->propertyValues[$name])) {
+            return;
```

Comment:
Had to put this here in order to be able to save nodes again... So I guess just throwing another exception would also not work... I'll try to find another solution.